app/src/main/java/org/dieschnittstelle/mobile/android/skeleton/TaskDetailViewActivity.java

```diff
@@ -78,13 +86,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         this.viewModel.isTaskOnDelete().observe(this, onDelete -> {
             if (onDelete) {
-                Intent returnIntent = new Intent();
-                returnIntent.putExtra(TASK_DETAIL_VIEW_KEY, task);
-                this.setResult(TaskDetailViewActivity.RESULT_OK, returnIntent);
-                this.finish();
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setView(android.R.layout.select_dialog_item);
+                builder.setMessage("Delete task?");
```

Comment:
wir haben das Task als Mission in UI genannt. Bitte vereinheitlichen

Reply:
erledigt